Form-Controls/index.html

```diff
@@ -13,15 +13,59 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+          <fieldset>
+            <legend>Please Enter your account</legend>
+            <label for="name">Name</label>
+            <input id="name" type="text" name="name" placeholder="Enter Your Full Name" required>
+            <label for="email"></label>
+            <input id="email" type="email" name="email" placeholder="Enter Your Email" required>  
+          </fieldset>
+          <fieldset>
+            <legend>Choose your T-Shirt</legend>
+            <hr> 
+            Colour:          
+              <div>
+                <label for="red">Red</label>
+                <input type="radio" name="Colour" id="red">
+              </div>
+              <div>
+                <label for="white">White</label>
+                <input type="radio" name="Colour" id="White">
+              </div>
+              <div>
+                <label for="Yellow">Yellow</label>
+                <input type="radio" name="Colour" id="Yellow">
+              </div>
+              <hr>
+              Size:
+              <label for="size"></label>
+              <select name="size of the T-Shirt" id="size" required>
+                <option value="Extra Small">XS</option>
+                <option value="Small">S</option>
+                <option value="Medium">M</option>
+                <option value="Large">L</option>
+                <option value="Extra-Large">XL</option>
+                <option value="2*Large">XXL</option>
```

Comment:
Please, use only lowercase letters and hyphens as values for your  attributes. `value` "2*Large" is not good, a better option is "xxl" or "extra-extra-large"